packages/web/src/scripts/util.js

```diff
@@ -580,17 +607,44 @@ function getEndPositionOfMatch(match) {
  * @returns The location of the text on Monaco editor by describing its column and line number range
  */
 export function findMonacoLocationOfJSONText(jsonPath, text, textModel) {
-    const results = findJSONLocationOfMonacoText(text, textModel)
-    let monacoLocation = {}
+    const results = findJSONLocationOfMonacoText(text, textModel);
+    let monacoLocation = {};
 
     if (results) {
-        results.forEach(result => {
+        results.forEach((result) => {
             if (jsonPath.localeCompare(result.path) === 0) {
-                monacoLocation = result.match.range
-                return
+                monacoLocation = result.match.range;
+                return;
             }
-        })
+        });
     }
 
-    return monacoLocation
+    return monacoLocation;
 }
+
+/**
+ * Checks if the input is a TD or TM
+ * @param {jsonDocument} 
+ * @returns "tm" or "td"
```

Comment:
Would it make sense to report a unknown value also so the caller can decide what it is defaulting to?